src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,282 @@
+| PipelineDescription.Version   | OPTIONAL. Version of the pipeline.                                                                                                                                                                                                           |
+| PipelineDescription.CodeURL   | OPTIONAL. URL where the code for the analysis can be found.                                                                                                                                                                                  |
+| PipelineDescription.Container | OPTIONAL. Object specifying the location and relevant attributes of software container image used to produce the derivative. Valid fields in this object include `Type`, `Tag` and `URI`.                                                    |
+| SourceDatasets                | OPTIONAL. A list of objects specifying the locations and relevant attributes of all source datasets. Valid fields in each object include `URL`, `DOI`, and `Version`.                                                                        |
+
+Example:
+
+```JSON
+{
+    "Name": "FMRIPREP Outputs",
+    "BIDSVersion": "TODO (depends when this PR will be merged)",
+    "PipelineDescription": {
+        "Name": "FMRIPREP",
+        "Version": "1.2.5",
+        "Container": {
+            "Type": "docker",
+            "Tag": "poldracklab/fmriprep:1.2.5"
+            }
+        },
+    "SourceDatasets": [
+        {
+            "DOI": "10.18112/openneuro.ds000114.v1.0.1",
+            "URL": "https://openneuro.org/datasets/ds000114/versions/1.0.1",
+            "Version": "1.0.1"
+        }
+    ]
+}
+```
+
+## Coordinate systems
+
+The coordinate system (a.k.a. space) a particular derivative is in SHOULD be
+denoted using a filename keyword `space` whenever such keyword is present in
+the filename template of a given derivative type. The allowed values for this
+keyword depend are identifiers given in section [Image-Based Coordinate
+Systems](../99-appendices/08-coordinate-systems.md#image-based-coordinate-systems).
+
+| File format                  | Description             |
+| ---------------------------- | ----------------------- |
+| NIfTI (`.nii` and `.nii.gz`) | Volume data             |
+| GIFTI (`.gii`)               | Surface data            |
+| CIFTI (`.nii`)               | Volume and surface data |
+
+Examples:
+
+```Text
+sub-01/func/sub-01_task-rest_space-MNI305_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-individual_bold.nii.gz
+sub-01/anat/sub-01_hemi-L_space-fsaverage5_thickness.shape.gii
+sub-01/anat/sub-01_hemi-R_space-fsaverage5_thickness.shape.gii
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.shape.gii
+sub-01/anat/sub-01_hemi-R_space-individual_thickness.shape.gii
+sub-01/func/sub-01_task-rest_space-HCPMNIfsLR32k_bold.nii
```

Comment:
`HCPMNIfsLR32k` does not exist (https://1891-150465237-gh.circle-artifacts.com/0/site/99-appendices/08-coordinate-systems.html#standard-template-identifiers). This should be either
```suggestion
sub-01/func/sub-01_task-rest_space-fsLR_bold.dtseries.nii
```
or
```suggestion
sub-01/func/sub-01_task-rest_space-MNI152NLin6Asym_res-32k_bold.nii.gz
```
I would prefer the former as introducing `res-` here would be problematic, and also I think the example was thought out to be a CIFTI file.